.NET/Microsoft.Recognizers.Definitions.Common/English/DateTimeDefinitions.cs

```diff
@@ -69,6 +69,10 @@ public static class DateTimeDefinitions
       public const string TimeTokenPrefix = @"at ";
       public const string TokenBeforeDate = @"on ";
       public const string TokenBeforeTime = @"at ";
+      public const string HalfTokenRegex = @"^(half)";
+      public const string QuarterTokenRegex = @"^((a\s+)?quarter)";
+      public const string ThreeQuarterTokenRegex = @"^(three\s+quarter)";
```

Comment:
Shouldn't this also include the plural form?

Reply:
right, added.